thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/AutotuneConfigDTO.java

```diff
@@ -1,11 +1,53 @@
 package com.linkedin.thirdeye.datalayer.dto;
 
 import com.linkedin.thirdeye.datalayer.pojo.AutotuneConfigBean;
+import com.linkedin.thirdeye.detector.email.filter.AlertFilter;
+import com.linkedin.thirdeye.detector.email.filter.BaseAlertFilter;
+import com.linkedin.thirdeye.detector.email.filter.DummyAlertFilter;
+import java.util.Properties;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 
+/**
+ * Auto Tune Configuration DTO. Given Alert filter, add alert filter setting to autotune config properties
+ * When autotune being triggered, autotuneConfig will be passed to tuning steps
+ */
 public class AutotuneConfigDTO extends AutotuneConfigBean {
   private static final Logger LOGGER = LoggerFactory.getLogger(AutotuneConfigDTO.class);
+  private AlertFilter alertFilter = new DummyAlertFilter();
+  private Properties tuningProps = new Properties();
```

Comment:
should this be a field in AutotuneConfigBean?

Reply:
It's to be decided. AutotuneConfigBean contains entities to be wrote to db. Not sure if these tuning related properties need to put to bean.